komodo-core/src/main/java/org/komodo/repository/RepositoryImpl.java

```diff
@@ -1323,6 +1323,16 @@ public KomodoObject komodoSearches(UnitOfWork transaction) throws KException {
     }
 
     @Override
+    public KomodoObject komodoServersNode(UnitOfWork transaction) throws KException {
+        ArgCheck.isNotNull( transaction, "transaction" ); //$NON-NLS-1$
+        ArgCheck.isTrue( ( transaction.getState() == org.komodo.spi.repository.Repository.UnitOfWork.State.NOT_STARTED ),
+        "transaction state is not NOT_STARTED" ); //$NON-NLS-1$
+
+        komodoEnvironment(transaction);
+        return create(transaction, SERVERS_ROOT, Environment.SERVERS);
```

Comment:
I think most if not all of the other grouping nodes currently do not have types defined, so probably not needed on the grouping node?